conanfile.py

```diff
@@ -54,6 +54,14 @@ class grpcConan(ConanFile):
         "re2/20201101"
     )
 
+    def build_requirements(self):
+        self.build_requires("protobuf/3.13.0")
+
+        # When cross building the recipe depends on itself to provide the plugins
+        if tools.cross_building(self.settings):
+            self.build_requires(
+                f"{self.name}/{self.version}@{self.user}/{self.channel}")
```

Comment:
While in principle Conan is still supporting Python 2, supporting Python 2 in recipes would be ideal. But at the very minimum, it has to support Python 3.5+

Please don't use f-strings

Reply:
Sure! I'm just so used to being able to use them :)